module-ballerina-openapi/annotation.bal

```diff
@@ -21,13 +21,17 @@
 # + excludeTags - Disable the OpenAPI validator for these tags
 # + excludeOperations - Disable the OpenAPI validator for these operations
 # + failOnErrors - Enable the OpenAPI validator
+# + autoEmbedToService - Enable auto-inject of OpenAPI documentation to current service
+# + autoGen - Enable OpenAPI documentation generation for current service
 public type ServiceInformation record {|
     string contract = "";
     string[]? tags = [];
     string[]? operations = [];
     string[]? excludeTags = [];
     string[]? excludeOperations = [];
     boolean failOnErrors = true;
+    boolean autoEmbedToService = true;
```

Comment:
Can this be `embedToService` or simply `embed`

Reply:
Addressed in [712ae126126dae27896b5f1eed5f0e0a063c2a07](https://github.com/ballerina-platform/ballerina-openapi/pull/599/commits/712ae126126dae27896b5f1eed5f0e0a063c2a07).